manifests/a/Autodesk/AutoCAD2022/2022/Autodesk.AutoCAD2022.installer.yaml

```
@@ -0,0 +1,15 @@
+# Created using wingetcreate 1.9.3.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.9.0.schema.json
+
+PackageIdentifier: Autodesk.AutoCAD2022
+PackageVersion: "2022"
+InstallerType: exe
+InstallerSwitches:
+  Silent: -q
+  SilentWithProgress: -q
+Installers:
+- Architecture: x64
+  InstallerUrl: http://dds.autodesk.com/NetSWDLD/2022/ACD/1E7D4EF7-A28E-3D3E-BA3C-C6FAE4AAB2E0/SFX/AutoCAD_2022_English_Win_64bit_dlm.sfx.exe
+  InstallerSha256: F7C886D879BD311635EBF229B9F5D9B6CE2171C668E905EE83C239FE9BBF2FB9

```

```suggestion
Installers:
- Architecture: x64
  InstallerLocale: en-US
  InstallerUrl: http://dds.autodesk.com/NetSWDLD/2022/ACD/1E7D4EF7-A28E-3D3E-BA3C-C6FAE4AAB2E0/SFX/AutoCAD_2022_English_Win_64bit_dlm.sfx.exe
  InstallerSha256: F7C886D879BD311635EBF229B9F5D9B6CE2171C668E905EE83C239FE9BBF2FB9
```

Is there an installer for this package in another language? If so, I suggest this modification.